python/dgl/backend/tensorflow/tensor.py

```diff
@@ -6,13 +6,12 @@
 import tensorflow as tf
 from tensorflow.python.eager import context
 import builtins
-import tfdlpack
 import numpy as np
-from tfdlpack import to_dlpack, from_dlpack
 
 from ... import ndarray as nd
 from ... import kernel as K
 from ...function.base import TargetCode
+from ...container import _api_internal
 
 TF_VERSION = LooseVersion(tf.__version__)
```

Comment:
Can we check the TF version and throw warning to users? We might want to check MX and TH versions too.

Reply:
Sure